components/Group/My-Groups.js

```diff
@@ -20,27 +20,30 @@ export default function MyGroups() {
   }, []);
 
   return (
-    <div className="flex flex-row justify-center gap-5 pt-5">
-      {groups?.map((group) => (
-        <Link href={"/group/" + `${group.id}`}>
-          <div className="flex flex-row bg-white gap-5 p-3 rounded-lg">
-            <Image
-              src={group.picture}
-              alt="group photo"
-              width={150}
-              height={150}
-            />
-            <div className="flex flex-col">
-              <span>
-                {group.name}
-              </span>
-              <span>
-                {group.location}
-              </span>
-            </div>
+    <div className="flex flex-col px-10 py-5 border-b-[1px] border-b-gray-500 ">
+      <h1 className="font-bold text-2xl self-center">My Groups</h1>
+      <div className="flex flex-row justify-center gap-5 pt-5">
+        {groups?.map((group, index) => (
+          <div className="flex flex-row bg-white gap-5 p-3 rounded-lg w-fit h-fit" key={index}>
```

Comment:
**Notes**: We will want to put the css elements in the standard order during UI clean up.